pet-api/pet-api/LogFiles/file-2022-11-26.txt

```diff
@@ -0,0 +1,14 @@
+Now listening on: https://localhost:7071
```

Comment:
add logs to the gitignore

Reply:
There are some problems with it, but I`ll try to find a possibility to resolve